data/wanderers start.txt

```diff
@@ -2443,6 +2443,7 @@ mission "Wanderers Ap'arak 3: Pug"
 		ship "Pug Arfecta" "Hork Ekekek Ptui"
 		ship "Pug Arfecta" "Graak Sput Kchoo"
 	on complete
+		dialog `Shortly after destroying the Pug ships, you received a message from Sobari Tele'ek: "We have received [news, reports] that you are destroying the ships of our Keeper [friends, allies] in the Ap'arak system. This is [unacceptable, intolerable]. From now on, you are no longer [welcome, allowed] in our [territory, space]."`
```

Comment:
Five multiple-meaning blocks in three (almost two) sentences seems high - what's the average use in the missions around it?

I think reducing it to 2 or 3 would be much more palatable. Perhaps `Keeper []` to `Keepers`, `[unacceptable, intolerable]` to `unforgivable`, and `in our [territory, space]` to `here`.

Doesn't have to be those exactly.

Reply:
What about this:

"We have received [news, reports] that you are destroying the ships of our Keepers in the Ap'arak system. This is unacceptable. From now on, you are no longer welcome in our [territory, space]."

Reply:
I'm gonna commit it then